arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -11,9 +11,12 @@
 #include <arch/arm/cortex_m/cmsis.h>
 #include <arch/arm/cortex_m/mpu/arm_mpu.h>
 #include <arch/arm/cortex_m/mpu/arm_core_mpu.h>
-#include <logging/sys_log.h>
 #include <linker/linker-defs.h>
 
+#define LOG_MODULE_NAME arm_mpu
```

Comment:
i wonder if there is point to create to independent instances (arm_mpu and arm_core_mpu). If they are always working together then it can be just arm_mpu (see http://docs.zephyrproject.org/subsystems/logging/logger.html#usage for details how to declare a multi-file module).